source/mfs-painters/multiframepainter/PostprocessingStage.cpp

```diff
@@ -122,8 +122,8 @@ void PostprocessingStage::generateNoiseTexture()
     auto texture = new globjects::Texture(gl::GL_TEXTURE_2D);
     texture->setParameter(gl::GL_TEXTURE_MIN_FILTER, gl::GL_NEAREST);
     texture->setParameter(gl::GL_TEXTURE_MAG_FILTER, gl::GL_NEAREST);
-    texture->setParameter(gl::GL_TEXTURE_WRAP_S, gl::GL_MIRRORED_REPEAT);
-    texture->setParameter(gl::GL_TEXTURE_WRAP_T, gl::GL_MIRRORED_REPEAT);
+    texture->setParameter(gl::GL_TEXTURE_WRAP_S, gl::GL_REPEAT);
```

Comment:
![ssao_mirrored_artifacts](https://cloud.githubusercontent.com/assets/1891915/14779617/35b9bf44-0ad9-11e6-97cb-37c2d9667540.png)

those visible borders of the noise texture are caused by the mirroring and go away with this change. it's probably only necessary for the noise texture and possibly for the reflection kernel (don't know exactly how that works), but i applied it to all textures for good measure.